app/retail/templates/home/nav.html

```diff
@@ -100,12 +100,16 @@
       {% if user.is_authenticated %}
         {% include 'shared/nav_auth.html' with source='authed' %}
       {% else %}
+        <a href="{% url 'social:begin' 'twitter' %}?next=/" class="nav-link interior">
+          <i class="fab fa-github mr-1" role="aria-hidden"></i>
```

Comment:
i dont think we need this login link in the nav.  thanks tho!

Reply:
Oh I see.

Reply:
can u pls remove

Reply:
already updated.